src/app/pages/quickMaps/pages/baselineDetails/mapView/mapView.component.ts

```diff
@@ -246,24 +357,24 @@ export class MapViewComponent implements AfterViewInit {
       this.thresholdMap.removeControl(this.thresholdLegend);
     }
 
-    // eslint-disable-next-line arrow-body-style
-    this.thresholdDataLayer = this.createGeoJsonLayer((feat: GeoJSON.Feature) => {
-      return this.getColourThreshold(feat.properties.mnThreshold);
-    }).addTo(this.thresholdMap);
+    this.thresholdDataLayer = this.createGeoJsonLayer((feat: GeoJSON.Feature) =>
+      this.getColourRange(null, feat.properties.mnThreshold, this.defaultColourScheme)).addTo(this.thresholdMap);
 
     this.thresholdLegend = new L.Control({ position: 'bottomright' });
 
     this.thresholdLegend.onAdd = () => {
       const div = L.DomUtil.create('div', 'info legend');
-      const range = [0, 20, 40, 60, 80, 99];
+      const range = [0, 10, 20, 40, 60, 80, 99];
```

Comment:
Be nicer if these were closer to the declaration of the colours, maybe in external class(es) too